Patches/Vanilla Races Expanded - Sanguophage/HediffDefs/HediffDef_SharpTalons.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Vanilla Races Expanded - Sanguophage</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/HediffDef[defName="VRE_SharpTalons"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>sharp talons</label>
+								<capacities>
+									<li>Cut</li>
+								</capacities>
+								<power>15</power>
```

Comment:
I'd reduce the damage to 6 - 8 and increase the AP to compensate a bit (given their 'origin', I think vampires should be able to cut their medieval-level armor to some extent). The Strigoi xenotype, which has the talons, also has a 125% melee damage factor. Combined with any decent level of Melee skill, the talons can readily one-shot bodyparts like limbs or necks.